Configs/.local/share/bin/swwwallpaper.sh

```diff
@@ -98,7 +98,6 @@ done
 swww query &> /dev/null
 if [ $? -ne 0 ] ; then
     swww-daemon --format xrgb &
-    swww query && swww restore
```

Comment:
why remove this ?

Reply:
I just copy pasted the file didnt know this will revert